### ISSUE TITLE
Farm PokeStops capturing Pokemons

### ISSUE THREAD

**Issue by RedSalsicha** [finndev/PokeBuddy#681]
When I turn on the "Farm PokeStops" he continues capturing Pokemons.
This happened after the last update, which has enabled the incubator

**Comment by curtiola** [comment]
That sounds like a good thing, do you not want to catch pokemon and have "catch pokemon" unchecked?

**Comment by RedSalsicha** [comment]
Yes, once when only were marked the "Farm PokeStops" he would not catch Pokemons, I was just walking and picking up items in pokestops.

Now only marking this field, it also catches Pokemons. =/

**Comment by curtiola** [comment]
Huh, strange, at least it will stop when you're pokemon bag is full I guess lol

I tried it too @finndev I can confirm this is a bug
